.github/workflows/AI.yml

```diff
@@ -71,7 +71,11 @@ jobs:
 
           summary_prompt = (
               f"Analyze the following pull request diff and summarize the changes. "
+              f"- Key files and components modified. "
```

Comment:
The summary prompt is becoming overly verbose. Consider consolidating the points to maintain clarity and conciseness.